Templates/HttpTrigger-Powershell/run.ps1

```diff
@@ -0,0 +1,8 @@
+$name = Get-Content $Env:req
```

Comment:
We wanted the Run button from portal to give a result with the request body. Originally the Node and C# were written this way as well. But we changed them.